OSDOCS-5442: add info on lvmd and ovn yaml files (#57324)
openshift/openshift-docs: modules/microshift-lvms-deployment.adoc

[id="lvms-deployment_{context}"]
= LVMS Deployment

LVMS is automatically deployed on to the cluster in the `openshift-storage` namespace after {product-title} starts.

LVMS uses `StorageCapacity` tracking to ensure that pods with an LVMS PVC are not scheduled if the requested storage is greater than the free storage of the volume group. For more information about `StorageCapacity` tracking, read link:https://kubernetes.io/docs/concepts/storage/storage-capacity/[Storage Capacity].

[id="microshift-lvms-setting-path_{context}"]
== Setting the LVMS path

> The `lvmd.yaml` file for the LMVS must be written to the same directory as the {product-title} `config.yaml` file. When {product-title} runs, the `/etc/microshift/lvmd.yaml` directory and file are accessed.

Reviewer: This could be clearer. I think the important detail is that the LVMD config file goes in `/etc/microshift/lvmd.yaml`. In _older_ versions, it could live next to the MicroShift config file in one of several places, but we only look in `/etc/microshift` for it now, so the first sentence doesn't add anything.

```suggestion
When {product-title} runs, it uses LVMS configuration from `/etc/microshift/lvmd.yaml`, if provided.
```

Author: Done. Do we need to give folks a command to create the file?

LVMS is automatically deployed on to the cluster in the `openshift-storage` namespace after {product-title} boots.

> //Q: is this correct, or should it be `microshift-namespace`?

Reviewer: It is `openshift-storage`.

> //Q: what's the procedure to create the file since it is not created by default?

Reviewer: Similar to elsewhere, something like

```
sudo cat >/etc/microshift/lvmd.conf <<EOF
socket-name: /run/topolvm/lvmd.sock
device-classes:
  - name: default
    volume-group: microshift
    spare-gb: 10
    default: true
EOF
```
